app/(main)/login/page.test.tsx

```diff
@@ -78,4 +78,16 @@ describe('Login', () => {
 
     mockUseAuthContext.isSignedIn = false;
   });
+
+  test('redirects to /league/all when user navigates to /login', async () => {
```

Comment:
Can you add a test that checks that `if user is logged in and on /league/all or any other page only accessible when logged in` and they try to go to `/login` or `/register` it redirects them to `/league/all`?